src/app/components/authorized/feed/feed.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BehaviorSubject } from 'rxjs';
+import { PostService } from 'src/app/services/post.service';
+import { FeedComponent } from './feed.component';
+
+export interface Post {
+    id: number;
+    title: string;
+    date_created: Date;
+    subreddit: string;
+    author: string;
+    commentCount: number;
+    post_type: string;
+    posted_url?: string;
+    description: string;
+}
```

Comment:
Don't you have that interface created as type somewhere in the app already?

Reply:
Yes.